WooCommerce/Classes/POS/Presentation/TotalsView.swift

```diff
@@ -249,24 +254,39 @@ private extension TotalsView {
 
     @ViewBuilder
     private var paymentsActionButtons: some View {
-        if posModel.paymentState == .cardPaymentSuccessful {
-            if isShowingPaymentsButtonSpacing {
-                Spacer().frame(height: Constants.paymentsButtonSpacing)
-            }
-            sendReceiptButton
-                .renderedIf(shouldShowSendReceiptButton)
-            newOrderButton
-                .onAppear {
-                    isShowingPaymentsButtonSpacing = false
-                    withAnimation(.default.delay(Constants.paymentsButtonButtonSpacingAnimationDelay)) {
-                        isShowingPaymentsButtonSpacing = true
+        ZStack {
```

Comment:
Lets leave it like in this in this scope of this PR, likely it will make sense to move it to success view p1733389827659689/1733375209.052419-slack-C070SJRA8DP.

We can see that all this logic changes when the success view logic changes.